econml/tests/test_automated_ml.py

```diff
@@ -45,10 +45,10 @@
     AutomatedKernelDML = addAutomatedML(KernelDML)
     AutomatedNonParamDML = \
         addAutomatedML(NonParamDML)
-    AutomatedForestDML = addAutomatedML(ForestDML)
+    AutomatedCausalForestDML = addAutomatedML(CausalForestDML)
 
     AUTOML_SETTINGS_REG = {
-        'experiment_timeout_minutes': 1,
+        'experiment_timeout_minutes': 15,
```

Comment:
15 minutes is the new lowest allowed value for the experiment timeout.

Reply:
Wow, that’s a lot